python/extensions.bzl

```diff
@@ -24,15 +24,27 @@ load("@rules_python//python/private:toolchains_repo.bzl", "get_host_os_arch", "g
 def _python_impl(module_ctx):
     for mod in module_ctx.modules:
         for toolchain_attr in mod.tags.toolchain:
-            python_register_toolchains(
-                name = toolchain_attr.name,
-                python_version = toolchain_attr.python_version,
-                bzlmod = True,
-                # Toolchain registration in bzlmod is done in MODULE file
-                register_toolchains = False,
-                register_coverage_tool = toolchain_attr.configure_coverage_tool,
-                ignore_root_user_error = toolchain_attr.ignore_root_user_error,
-            )
+            if toolchain_attr.base_url:
```

Comment:
@rickeylev not sure if there is a cleaner way to do this.  If we don't set the base_url when we call python.toolchain, we end up with an empty string passed in.  Then python_register_toolchains passes in an empty value to python_repository.  We may have a weird use case where an empty URL is allowed; if not, I can test for an empty URL in python_register_toolchains, which is fewer lines of code.